debian/changelog

```diff
@@ -22,7 +22,7 @@ ubuntu-advantage-tools (27.3~21.10.1) impish; urgency=medium
       + print warning about generic fips kernel if cloud-id fails
     - cloud:
       + rely only on cloud-id to determine cloud type (LP: #1940131)
-      + catch errors when determining cloud type (LP: #1938207) (GH: #1541)
+      + catch errors when determining cloud type (LP: #1938207, LP: #1944676) (GH: #1541)
```

Comment:
damn this was my bad as well, I included this in the SRU bug and forgot about changelog :|

Reply:
All is good, I saw the addition to the bug and forgot to add it until now :)